ecospheres/migrations/20250424_1_logistique_remove_subtheme.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from minicli import cli, run
+
+from ecospheres.api import DatagouvfrAPI
+
+TAG_PREFIX = "logistique-subtheme"
+
+
+@cli
+def migrate_bouquets(slug: str = "", dry_run: bool = False, env: str = "ecospheres/migrations/data/logistique/env.yaml"):
+    """
+    Remove "subtheme" tags from bouquets.
+    """
+    api = DatagouvfrAPI(env)
+    bouquets = api.get_bouquets()
+    bouquets = [b for b in bouquets if b["slug"] == slug] if slug else bouquets
+    for bouquet in bouquets:
+        print(f"--> Handling {bouquet['slug']}...")
+        if not any(t.startswith(TAG_PREFIX) for t in bouquet["tags"]):
+            print("No subtheme tag to remove, skipping.")
+            continue
+        tags = [t for t in bouquet["tags"] if not t.startswith(TAG_PREFIX)]
```

Comment:
Nitpick: Reads better?

```
filtered_tags = [t for t in bouquet["tags"] if not t.startswith(TAG_PREFIX)]
if filtered_tags == tags:
    print("No subtheme tag to remove, skipping.")
    continue

payload = {"tags": filtered_tags}
```

Reply:
It's a copy from the remove chantiers one, so I'll leave it like that :-)